object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/ObjectStoreClient.scala

```diff
@@ -26,33 +26,42 @@ import scala.language.higherKinds
 
 class ObjectStoreClient[REQ, RES](client: HttpClient[REQ, RES], config: ObjectStoreClientConfig) {
 
+  private val authorizationHeader: List[(String, String)] =
```

Comment:
It might be clearer if `authorizationHeader = "Authorization -> config.authorizationToken` - i.e. a single one, and then is wrapped in List when used - e.g. `client.delete(s"$url/object/$location", List(authorizationHeader))`